.eslintignore

```diff
@@ -31,4 +31,8 @@ examples/native/ios
 examples/native/android
 examples/native/metro.config.js
 examples/crna-kitchen-sink/metro.config.js
-storybook.requires.js
\ No newline at end of file
+storybook.requires.js
+scripts/
```

Comment:
If you can brief me a bit on the background of these excludes, then it would be awesome :+1: . `prettier.config.js` and `tsconfig.json` seem to lint OK as-is, even if we don't ignore them 🤔 .

`scripts/` excludes both `/scripts` and `app/react-native/scripts`. Both have files with `/* eslint-disable */`s, which looks a bit confusing (as they are now excluded from linting anyhow). Instead of excluding both `/scripts` directories completely, could we fix the eslint errors?

Made a small PR, https://github.com/storybookjs/react-native/pull/209 , that contains this & a simple fix for the regex thing we discussed in https://github.com/storybookjs/react-native/pull/182#discussion_r667504748

Reply:
I was battling with a lot of linting issues so this is probably just a result of that. Any improvements are welcomed so thanks for the fix 🙏 . Merged your pr already.